src/main/java/org/interledger/btp/BilateralTransferProtocolRuntimeException.java

```diff
@@ -0,0 +1,68 @@
+package org.interledger.btp;
+
+public class BilateralTransferProtocolRuntimeException extends RuntimeException {
+
+  /**
+   * Constructs a new BTP runtime exception with the specified detail message.
+   * The cause is not initialized, and may subsequently be initialized by a
+   * call to {@link #initCause}.
+   *
+   * @param message the detail message. The detail message is saved for
+   *                later retrieval by the {@link #getMessage()} method.
+   */
+  public BilateralTransferProtocolRuntimeException(String message) {
```

Comment:
I think we might want an optionally-present transferId here....most exception will have that, correct?